cloud/shared/bin/lib/terraform.sh

```diff
@@ -2,7 +2,7 @@
 
 # https://www.baeldung.com/linux/store-command-in-variable
 readonly TERRAFORM_CMD=("terraform" "-chdir=${TERRAFORM_TEMPLATE_DIR}")
-readonly TERRAFORM_APPLY=(${TERRAFORM_CMD[@]} "apply" "-input=false" "-var-file=${TF_VAR_FILENAME}")
+readonly TERRAFORM_APPLY=(${TERRAFORM_CMD[@]} "apply" "-input=false" "-json")
```

Comment:
question (blocking): TERRAFORM_APPLY doesn't need the var file?

Reply:
the plan does now; i split it up to use plan which allows the test to work but also might allow for future automation and like approvals to deploy (which is what tf documentation about how to handle automation recommends)